lib/thredded/truthy_hash.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module Thredded
+  module TruthyHash
```

Comment:
Looks like we want to track overrides of the default values here.
But this approach doesn't allow for changing the default for a certain type of notification to false.

Perhaps a more specific `Thredded::NotificationOptions` class that validates the keys and serializes non-default values would be better?